src/components/Ratings/ReviewTile.jsx

```diff
@@ -48,6 +63,8 @@ const ReviewTile = ({ review }) => {
 
   return (
     <Tile>
+      { !reported ?
```

Comment:
Instead of having a either/or ternary operator here (using a ? and : ), you can conditionally render one item
{[truthy if statement here] && [whatever you want to conditionally render]}